redaxo/src/core/lib/util/path_default_provider.php

```diff
@@ -79,7 +79,7 @@ public function frontendController()
     public function backend($file)
     {
         if ($this->provideAbsolutes) {
-            return $this->base($this->backend . '/' . $file);
+            return $this->frontend($this->backend . '/' . $file);
```

Comment:
Agree. -> phpdoc mit konkreten beispielen (wie in gregor comment) sollten da helfen